packages/config-loader/src/loader.test.ts

```diff
@@ -29,8 +31,20 @@ describe('loadConfig', () => {
       '/root/app-config.development.yaml': `
         app:
           sessionKey: development-key
+        backend:
+          $include: ./included.yaml
+        other:
+          $include: secrets/included.yaml
       `,
       '/root/secrets/session-key.txt': 'abc123',
+      '/root/secrets/included.yaml': `
+        secret:
+          $file: session-key.txt
+      `,
+      '/root/included.yaml': `
+        foo:
+          bar: token \${MY_SECRET}
```

Comment:
Since it's a backtick string (starting on line 44) - ts also uses `${`, hence it needs to be escaped so the compiler doesn't see it and try to find a variable named MY_SECRET :)